### DISCUSSION TITLE
Style Select dropdown

### DISCUSSION THREAD

**Discussion by mlarcher** [mantinedev/mantine#1766] [Q&A]
Not sure if's that's an issue or if I'm doing it wrong, but I can't manage to add style to the Select dropdown component. 
Shouldn't something like that work ?
```jsx
<Select
      {...otherProps}
      data={data}
      sx={(theme) => ({
        dropdown: {
          backgroundColor: 'red',
        },
      })}
    />
```

**Comment by MR-Mostafa** [comment]
If `withinPortal` is `true`, you should use `styles` instead of `sx`.

**Reply by mlarcher** [reply]
thank you for the reply, but it is not the case

**Reply by MR-Mostafa** [reply]
It works correctly in this example؛
[codesandbox](https://codesandbox.io/s/confident-goldwasser-x4fzfi)

**Reply by mlarcher** [reply]
Oh, that means `withinPortal` is true by default. I didn't realize that. Thank you!

**Comment by rtivital** [comment]
You should use `styles` instead of `sx`:
```tsx
<Select
      {...otherProps}
      data={data}
      styles={(theme) => ({
        dropdown: {
          backgroundColor: 'red',
        },
      })}
    />
```
